### PR TITLE
T-Upgrade pinia

### PR DESCRIPTION
**Pull request by Ovsdrak**
Fixes: #number 

## Description

Replaces vuex stores for pinia stores
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [X] My code follows the code style of this project.
    - Good variable naming.
    - Constants are capitalized and snake cased.
    - Code uses `const` instead of `let` and `var`.
- [X] My commit history is nice and readable
- [X] My PR doesn't contain more than 10 or max 15 files changed
- [X] I've added the number of the issue I'm fixing at the top of my file
- [X] My code has been reviewed
- [ ] My code passed all the checks on the Pipeline (Action)
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: imports/startup/client/index.ts [hunk 1]
```diff
@@ -27,24 +25,13 @@ declare module 'vue' {
...
-	interface ComponentCustomProperties {
-		$store: Store<State>;
-	}
-}
-
 Meteor.startup(() => {
 	loadFonts();
 	const emitter = mitt();
+	const pinia = createPinia();
+	// @ts-ignore
```
Comment (diavrank) [inline imports/startup/client/index.ts:32]: is this needed?

File: imports/ui/App.vue [hunk 2]
```diff
@@ -18,6 +18,7 @@ import { defineComponent } from 'vue';
 export default defineComponent({
   name: 'App',
   components: { AlertMessage, Loader },
+
```
Comment (diavrank) [inline imports/ui/App.vue]:
```suggestion
```

File: imports/ui/layouts/shared/NavigationDrawer.vue [hunk 3]
```diff
@@ -50,6 +50,10 @@
 	export default defineComponent({
 		name: 'NavigationDrawer',
 		components: { FooterView },
+    setup() {
+      const temporalStore = useTemporalStore();
+      return { temporalStore };
+    },
```
Comment (diavrank) [inline imports/ui/layouts/shared/NavigationDrawer.vue]: linting

File: imports/ui/layouts/shared/NavigationDrawer.vue [hunk 4]
```diff
@@ -77,15 +77,18 @@
...
 			'$route'() {
 				this.updateSelectedOption();
 			},
-			drawer(newValue) {
-				this.navigationDrawer = newValue;
+			'temporalStore.drawer': {
+        immediate: true,
+        handler(newValue) {
+				  this.navigationDrawer = newValue;
+        }
```
Comment (diavrank) [inline imports/ui/layouts/shared/NavigationDrawer.vue]: linting

File: imports/ui/stores/auth.ts [hunk 5]
```diff
@@ -0,0 +1,45 @@
+import { defineStore } from 'pinia';
+import { useStorage } from '@vueuse/core';
```
Comment (diavrank) [inline imports/ui/stores/auth.ts:2]: is this the equivalent to vuex-persist?

File: imports/startup/client/index.ts (hunk 1 above)
Comment (Ovsdrak) [reply imports/startup/client/index.ts:32]: yes, it was throwing mistype error with the line 33. It was an existing line

File: imports/ui/stores/auth.ts (hunk 5 above)
Comment (Ovsdrak) [reply imports/ui/stores/auth.ts:2]: yes, it is

File: imports/ui/layouts/shared/NavigationDrawer.vue (hunk 4 above)
Comment (Ovsdrak) [reply imports/ui/layouts/shared/NavigationDrawer.vue]: Done [89bb4e8](https://github.com/diavrank/theory-swe/pull/8/commits/89bb4e8a75c19e67078e32445699957c3aab2dab)!

File: imports/ui/layouts/shared/NavigationDrawer.vue (hunk 3 above)
Comment (Ovsdrak) [reply imports/ui/layouts/shared/NavigationDrawer.vue]: Done [89bb4e8](https://github.com/diavrank/theory-swe/pull/8/commits/89bb4e8a75c19e67078e32445699957c3aab2dab)!

File: imports/ui/layouts/shared/NavigationDrawer.vue [hunk 6]
```diff
@@ -21,90 +21,90 @@
...
-			background-color: var(--v-accent-base)
+  .active
+    color: white
+    background-color: var(--v-accent-base)
 
-		.theme--dark.v-list-item--active:before
-			opacity: 0
+  .theme--dark.v-list-item--active:before
+    opacity: 0
 </style>
```
Comment (diavrank) [inline imports/ui/layouts/shared/NavigationDrawer.vue:110]: are these indentations correct? I think they classes should be stuck to the left (without any space). Are you using AntwareSchema.xml?

Comment (Ovsdrak) [reply imports/ui/layouts/shared/NavigationDrawer.vue:110]: they're ok, remember we are using SASS syntax, I'm using class nesting for that

**Pull request merged**